# Publish the battery as read from the FC
publish_fc_battery: true

# Use the initial heading as the zero point for all future headings
initial_heading_as_offset: true

# Calibrate the accelerometer every time the foot switches say
# the drone is on the ground
calibrate_accelerometer: false

# Timeout for determining validity of contact switch messages
valid_landing_detected_message_delay: 0.2

# Offset for timestamping orientation messages due to filter delay on the FC
imu_orientation_timestamp_offset: -0.00

# Publish transform
publish_orientation_transform: false

# Variances
imu_accel_x_var: 0.04
imu_accel_y_var: 0.04
imu_accel_z_var: 0.01

imu_gyro_x_var: 0.0025
imu_gyro_y_var: 0.0025
imu_gyro_z_var: 0.0025
